src/libcore/lib.rs

```diff
@@ -244,9 +244,6 @@ macro_rules! vector_impl { ($([$f:ident, $($args:tt)*]),*) => { $($f!($($args)*)
 #[cfg(not(stage0))] // allow changes to how stdsimd works in stage0
 mod coresimd;
 
-#[unstable(feature = "stdsimd", issue = "48556")]
-#[cfg(not(stage0))]
-pub use coresimd::simd;
```

Comment:
This removal broke the CI of rand, but I have trouble finding the reason. Is there an alternative to use simd with `no_std`, or is this no longer supported?